### ISSUE TITLE
Invalid glob argument when running in CI or pre-commit hook

### ISSUE THREAD

**Issue by edi9999** [jedmao/eclint#71]
The error is as follows : 

```
/opt/pclm/lib/ui/node_modules/vinyl-fs/lib/src/index.js:32
    throw new Error('Invalid glob argument: ' + glob);
    ^

Error: Invalid glob argument: 
    at Object.src (/opt/pclm/lib/ui/node_modules/vinyl-fs/lib/src/index.js:32:11)
    at Command._action (/opt/pclm/lib/ui/node_modules/eclint/dist/cli.js:79:22)
    at Command.parse (/opt/pclm/lib/ui/node_modules/gitlike-cli/lib/Command.js:284:18)
    at Command.tryParseSubcommand (/opt/pclm/lib/ui/node_modules/gitlike-cli/lib/Command.js:318:17)
    at Command.parse (/opt/pclm/lib/ui/node_modules/gitlike-cli/lib/Command.js:269:22)
    at Object.<anonymous> (/opt/pclm/lib/ui/node_modules/eclint/bin/eclint.js:4:5)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:427:7)
    at startup (bootstrap_node.js:151:9)
    at bootstrap_node.js:542:3
```

My pre-commit hook is just : 

```
#!/bin/bash

./node_modules/.bin/eclint check
```

It works well if I run this in a normal console.

After searching a bit, I found out that the problem comes from args.files not being empty : 

```
check.action(function (args, options) {
        console.log(args.files);
}
```

If I do that, I get "undefined" if I run it from a shell, but I get a big result when I run it from my pre-commit hook : 

```
[ Socket {
    connecting: false,
    _hadError: false,
    _handle:
     Pipe {
       bytesRead: 0,
       _externalStream: [External],
       fd: 0,
       writeQueueSize: 0,
       owner: [Circular],
       onread: [Function: onread],
       reading: false },
    _parent: null,
    _host: null,
    _readableState:
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: null,
       ended: false,
       endEmitted: false,
       reading: false,
       sync: false,
       needReadable: true,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: true,
    domain: null,
    _events:
     { end: [Object],
       finish: [Function: onSocketFinish],
       _socketEnd: [Function: onSocketEnd],
       pause: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    _writableState:
     WritableState {
       objectMode: false,
       highWaterMark: 16384,
       finalCalled: false,
       needDrain: false,
       ending: false,
       ended: true,
       finished: false,
       destroyed: false,
       decodeStrings: false,
       defaultEncoding: 'utf8',
       length: 0,
       writing: false,
       corked: 0,
       sync: true,
       bufferProcessing: false,
       onwrite: [Function: bound onwrite],
       writecb: null,
       writelen: 0,
       bufferedRequest: null,
       lastBufferedRequest: null,
       pendingcb: 0,
       prefinished: false,
       errorEmitted: false,
       bufferedRequestCount: 0,
       corkedRequestsFree: [Object] },
    writable: false,
    allowHalfOpen: false,
    _bytesDispatched: 0,
    _sockname: null,
    _pendingData: null,
    _pendingEncoding: '',
    server: null,
    _server: null,
    fd: 0,
    [Symbol(asyncId)]: 11,
    [Symbol(bytesRead)]: 0 } ]
```

I don't have a clue what "Sockets" have to do with that, but I think it is very likely a bug either in eclint or in git-like-cli.

**Comment by edi9999** [comment]
The easiest way to reproduce is with 

`cat | ./node_modules/.bin/eclint check`

**Comment by edi9999** [comment]
So the problem is that whenever the process has stdin, it will crash. 

This is not the case for most commands, for example :

echo "" | git status 

Works without a problem

**Comment by gucong3000** [comment]
LGM

**Comment by edi9999** [comment]
It seems very related to https://github.com/jedmao/eclint/pull/32

**Comment by gucong3000** [comment]
Fixed in #73
